src/App.vue

```diff
@@ -968,7 +997,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
       }
 
       // ensure types match
-      if (nrResponse.legalType !== this.getEntityType) {
+      if ((nrResponse.legalType as unknown as CorpTypeCd) !== this.getEntityType) {
```

Comment:
A bit of a hack to work around a type issue (due to different import specs for CorpTypeCd).